Core/clim-basic/clim-basic.asd

```diff
@@ -1,3 +1,5 @@
+(in-package #:asdf-user)
```

Comment:
I suggest doing "ASDF cleanup" in a separate commit for all defined systems after discussing which style we want to use.